### PR TITLE
Readme changes

### PR DESCRIPTION
**Pull request by rasapala**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,25 @@
----
-layout: default
-title: Home
-nav_order: 1
----
+# OpenVINO&trade; Model Server fork of [MediaPipe](https://google.github.io/mediapipe/).
+This repository is allowing users to take advantage of OpenVINO&trade; Model Serving in mediapipe examples. During the graph execution the inference calls on pretrained models are directed to model server instance initialized and parametrized by the specific inference calculators.
```
Comment (atobiszei) [inline README.md]:
```suggestion
This repository is allowing users to take advantage of OpenVINO&trade; Model Serving in mediapipe framework. During the graph execution the inference calls on pretrained models are directed to model server instance initialized and parametrized by the specific inference calculators.
```
We just show in several demos full required configuration but any graph with proper adjustements should be able to run with OVMS.

File: README.md [hunk 2]
```diff
@@ -25,47 +40,66 @@ nav_order: 1
...
+- build your new application using the bazel command inside the mediapipe_ovms container, for example in object_detection case it is:
+'build -c opt --define MEDIAPIPE_DISABLE_GPU=1 mediapipe/examples/desktop/object_detection:object_detection_openvino'
+- execute your application with the specific input parameters, for example in object_detection case those are:
+'bazel-bin/mediapipe/examples/desktop/object_detection/object_detection_openvino --calculator_graph_config_file mediapipe/graphs/object_detection/object_detection_desktop_openvino_graph.pbtxt --input_side_packets "input_video_path=/mediapipe/mediapipe/examples/desktop/object_detection/test_video.mp4,output_video_path=/mediapipe/tested_video.mp4"'
+- in object_detection example the input video will get all the objects detected by the model and return the tested_video.mp4 output.
+
+
+Below you can find the oryginal Mediapipe repository description.
```
Comment (atobiszei) [inline README.md]:
```suggestion
Below you can find the original Mediapipe repository description.
```

File: README.md (hunk 1 above)
Comment (rasapala) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (rasapala) [reply README.md]: Done.

File: docs/deployment.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Fast deployment instructions
```
Comment (dtrawins) [inline docs/deployment.md]:
```suggestion
# Quick start guide
```

File: README.md [hunk 4]
```diff
@@ -1,70 +1,40 @@
...
-```bash
-make run_pose_tracking
-```
-
-# Running face detection demo inside mediapipe_ovms container
-```bash
-make run_face_detection
-```
+# OpenVINO&trade; Model Server fork of [MediaPipe](https://google.github.io/mediapipe/).
+This repository is allowing users to take advantage of OpenVINO&trade; in mediapipe framework. During the graph execution the inference calls on pretrained models are directed to OpenVINO&trade; Model Server instance initialized and parametrized by the specific inference calculators.
```
Comment (dkalinowski) [inline README.md]:
```suggestion
This repository allows users to take advantage of OpenVINO&trade; in mediapipe framework. During the graph execution the inference calls on pretrained models are directed to OpenVINO&trade; Model Server instance initialized and parametrized by the specific inference calculators.
```

File: README.md [hunk 5]
```diff
@@ -1,70 +1,40 @@
...
+
+We show in several demos the full required configuration but any graph with proper adjustements can run with OpenVINO&trade; backend.
```
Comment (dkalinowski) [inline README.md]:
```suggestion
This repository shows multiple demos and its full configurations. It is also possible to create or adjust your own graphs to work with OpenVINO&trade; backend.
```

File: README.md [hunk 6]
```diff
@@ -1,70 +1,40 @@
...
+
+# List of main features
+- inference calculators that allow the use of OpenVINO&trade; backend for effective inference inside mediapipe framework.
```
Comment (dkalinowski) [inline README.md]:
```suggestion
- inference calculators allowing the use of OpenVINO&trade; backend for effective inference inside mediapipe framework.
```

File: README.md (hunk 5 above)
Comment (rasapala) [reply README.md]: Done.

File: README.md (hunk 4 above)
Comment (rasapala) [reply README.md]: Done.

File: README.md (hunk 6 above)
Comment (rasapala) [reply README.md]: Removed.

**Review by dtrawins — APPROVED** [review]

File: docs/deployment.md (hunk 3 above)
Comment (rasapala) [reply docs/deployment.md]: Done.

**Review by atobiszei — APPROVED** [review]

**Pull request merged**